Makefile

```diff
@@ -32,6 +32,11 @@ export GOPROXY
 # Active module mode, as we use go modules to manage dependencies
 export GO111MODULE=on
 
+# Kubebuilder.
+export KUBEBUILDER_ENVTEST_KUBERNETES_VERSION ?= 1.24.1
```

Comment:
One thing I'm not quite sure on is if we want to use this version or not yet.  Is there a source of truth somewhere in the repo?